decentraland/unity-renderer
# feat: add color picker for hair and eye color

## What does this PR change?

Fix #1743
Adds a color picker for eye and hair color.
The color picker is composed of 3 sliders for color, saturation and brightness.
...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/color-picker
2. Customize you character
3. Verify the color picker behavior for hair and eye color
4. Verify that it randomizes when using the random character function

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
